.github/bin/redshift/delete-aws-redshift.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+REDSHIFT_SCRIPTS_DIR="${BASH_SOURCE%/*}"
+
+if [[ ! -f "${REDSHIFT_SCRIPTS_DIR}/.cluster-identifier" ]];  then
+    echo "Missing file ${REDSHIFT_SCRIPTS_DIR}/.cluster-identifier"
+    exit 0
+fi
+
+REDSHIFT_CLUSTER_IDENTIFIER=$(cat $REDSHIFT_SCRIPTS_DIR/.cluster-identifier)
+
+echo "Deleting Amazon Redshift cluster $REDSHIFT_CLUSTER_IDENTIFIER"
+REDSHIFT_DELETE_CLUSTER_OUTPUT=$(aws redshift delete-cluster --cluster-identifier $REDSHIFT_CLUSTER_IDENTIFIER --skip-final-cluster-snapshot)
+
+if [ -z "${REDSHIFT_DELETE_CLUSTER_OUTPUT}" ]; then
+    # Only show errors
+    echo ${REDSHIFT_DELETE_CLUSTER_OUTPUT}
+    exit 1
+fi
+
+echo "Waiting for the Amazon Redshift cluster $REDSHIFT_CLUSTER_IDENTIFIER to be deleted"
+aws redshift wait cluster-deleted \
```

Comment:
Is there a timeout here? Do we have to wait at all? If this runs in a PR and fails, can the PR author do anything about the failure? If not, I'd log an error but not cause it to fail the whole pipeline.

Reply:
https://awscli.amazonaws.com/v2/documentation/api/2.1.29/reference/redshift/wait/cluster-deleted.html

> Wait until ClusterNotFound is thrown when polling with describe-clusters. It will poll every 60 seconds until a successful state has been reached. This will exit with a return code of 255 after 30 failed checks.

I've removed `-e`  from `set` to avoid exiting with `255` in such situations.